cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -103,14 +117,19 @@ describe('Script Builder', () => {
 
   before(() => {
     cy.flush().then(() => {
-      cy.signin().then(() => {
-        cy.get('@org').then(({id, name}: Organization) => {
+      return cy.signin().then(() => {
+        return cy.get('@org').then(({id, name}: Organization) => {
```

Comment:
For reasons behind some of these changes, I recommend https://docs.influxdata.io/ci/cypress-best-practices/